src/main/java/org/openhim/mediator/orchestration/RepositoryActor.java

```diff
@@ -66,7 +66,7 @@ public RepositoryActor(MediatorConfig config) {
     private void readMessage() {
         contentType = originalRequest.getHeaders().get("Content-Type");
 
-        if (contentType!=null && contentType.contains("multipart/related")) {
+        if (contentType!=null && (contentType.contains("multipart/related") || contentType.contains("multipart/form-data"))) {
```

Comment:
I saw that curl was using that type when submitting multipart (e.g. `curl nana -FxdsMetadata -Fdoc`), so it just made it easier when I was testing. But yeah it isn't strictly correct. Idk, seems okay to leave it; what do you think?